Modules/booking/View/Booking/simplecaldisplay.php

```diff
@@ -122,15 +123,15 @@
 					$last_end_time = mktime($day_begin,0,0,$temp[1], $temp[2], $temp[0]);
 					foreach($calData[$calDay][$resId] as $hcalEntry) { ?>
 						<?php
-						
+						$hcalEntry['client_name'] = ($hcalEntry['responsible_id'] > 0) ? $modelClient->getName($id_space, $hcalEntry['responsible_id']) : "";
```

Comment:
You make a request to get client name in a loop, for each entry. Could increase a lot number of requests.

I think client name could be fetched directly in cal entries via a join in request